test/rules/no-unbound-method/default/test.ts.lint

```diff
@@ -1,3 +1,18 @@
+class C {
+    constructor() {
+        this.foo = 1;
+        this.methodA = this.methodA.bind(this);
+    }
+    methodA() {}
+    methodB() {}
+}
+
+const c = new C();
+// OK if bound in constructor
+[0].forEach(c.methodA);
+[0].forEach(c.methodB);
+            ~~~~~~~~~ [0]
+
```

Comment:
Yes, you are absolutely right, this isn’t handled and also defeats [my argument](https://github.com/palantir/tslint/issues/3016#issuecomment-313842057). 😄

I will close the PR and issue then and need to either disable this rule or convert my code base to use arrow functions instead of `bind`. 😞